packages/flutter/lib/src/material/popup_menu_theme.dart

```diff
@@ -0,0 +1,156 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+import 'dart:ui' show lerpDouble;
+
+/// Defines default property values for [PopupMenuEntry] widgets.
+///
+/// Descendant widgets obtain the current [PopupMenuEntryThemeData] object
+/// using `Theme.of(context).popupMenuTheme`. Instances of
+/// [PopupMenuEntryThemeData] can be customized with
+/// [PopupMenuEntryThemeData.copyWith].
+///
+/// Typically a [PopupMenuEntryThemeData] is specified as part of the
+/// overall [Theme] with [ThemeData.popupMenuTheme].
+///
+/// All [PopupMenuEntryThemeData] properties are `null` by default.
+/// When null, the [PopupMenuEntry] will provide its own defaults.
+///
+/// See also:
+///
+///  * [ThemeData], which describes the overall theme information for the
+///    application.
+class PopupMenuThemeData extends Diagnosticable {
+  /// Creates a theme that can be used for [ThemeData.popupMenuTheme].
```

Comment:
This isn't entirely accurate, since this is the configuration used by [PopupMenuTheme].
```suggestion
  /// Creates the set of properties used to configure [PopupMenuTheme].
```

Reply:
Done